### DISCUSSION TITLE
ONNX version of open-ai/whisper large model

### DISCUSSION THREAD

**Discussion by Kirankumar2609** [onnx/onnx#4723] [Q&A]
Has anyone tried to export open-ai/whisper large model to ONNX and then use it for transcription/translation as mentioned in the below hf website.
https://huggingface.co/docs/transformers/serialization#configuration-based-approach

Need help in getting the prediction from whisper_model.onnx file

**Comment by xadupre** [comment]
Once you have an onnx file, you can use onnxruntime to compute the prediction (https://onnxruntime.ai/docs/). The documentation for the python binding is here: https://onnxruntime.ai/docs/api/python/tutorial.html.
